include/map_header_data.h

```diff
@@ -0,0 +1,91 @@
+#ifndef POKEPLATINUM_MAP_HEADER_DATA_H
+#define POKEPLATINUM_MAP_HEADER_DATA_H
+
+#include "constants/heap.h"
+#include "field/field_system_decl.h"
+#include "overlay006/struct_ov6_02242634.h"
+
+typedef struct BgEvent {
+    u16 script;
+    u16 type;
+    int x;
+    int z;
+    int y;
+    u16 playerFacingDir;
+    u8 padding[2];
+} BgEvent;
+
+typedef struct ObjectEvent {
+    u16 localID;
+    u16 graphicsID;
+    u16 movementType;
+    u16 trainerType;
+    u16 flag;
+    u16 script;
+    s16 dir;
+    u16 data[3];
+    s16 movementRangeX;
+    s16 movementRangeZ;
+    u16 x;
+    u16 z;
+    fx32 y;
+} ObjectEvent;
+
+typedef struct WarpEvent {
+    u16 x;
+    u16 z;
+    u16 destHeaderID;
+    u16 destWarpID;
+    u8 unused[4];
```

Comment:
Should this be a `u32`?

Reply:
What's the difference?

Reply:
Nothing, really, I guess. It's fine as is.